File: september/08-09/08-09.py

def build_acronym(s):

    words = s.split()
    ignore_words = ['a', 'for', 'an', 'and', 'by', 'of']
    acronym = ''

    for i in range(len(words)):
            
        word = words[i]
            
        if(i == 0):
            acronym += word[0].upper()
        else:
            if word not in ignore_words:
                acronym += word[0].upper()
                
    return acronym

# test
#print (build_acronym("for your information"))
assert build_acronym("for your information") == "FYI"

File: september/08-09/08-09.js
function buildAcronym(str) {

  let words = str.split(' ');
  const ignoreWords = ['a', 'for', 'an', 'and', 'by', 'of'];
  let acronym = '';

  for(let i=0; i < words.length; i++){
    
    let word = words[i];
    if(i == 0){
      acronym += word[0].toUpperCase();
    }else{
      if(!ignoreWords.includes(word)){
        acronym += word[0].toUpperCase();
      }
    }
  }  

  return acronym;
}